setup/README.md

```diff
@@ -248,7 +244,7 @@ Your signed images were allowed through, in contrast to those unsigned ones with
 
 ## Cleanup
 
-If you want to remove Connaisseur, run `make uninstall`.
+If you want to remove Connaisseur, run `make uninstall`. We do not delete the namespace, since you might have deployed things in there, which we would inadvertently delete. If you want to get rid of the namespace, run `kubectl delete ns WHATEVEROURDEFAULTIS`.
```

Comment:
```suggestion
If you want to remove Connaisseur, run `make uninstall`. We do not delete the namespace, since you might have deployed things in there, which we would inadvertently delete. If you want to get rid of the namespace, run `kubectl delete ns connaisseur` (or whatever other namespace you installed Connaisseur to).
```